Sources/TuistGenerator/Descriptors/ProjectDescriptor.swift

```diff
@@ -0,0 +1,27 @@
+import Basic
+import Foundation
+import XcodeProj
+
+public class ProjectDescriptor {
+    /// Path to the project
+    public var path: AbsolutePath
+
+    /// Path to the xcodeproj file
+    public var xcodeprojPath: AbsolutePath
+
+    public var xcodeProj: XcodeProj
+    public var schemes: [SchemeDescriptor]
```

Comment:
Nitpick: `schemeDescriptors`. Since we have `XCScheme`, `Scheme`, `SchemeDescriptor`, I think it's worth being more precise with the names that we use.